src/include/platform_support.h

```diff
@@ -28,8 +28,10 @@
 
 #if PC_HOSTED == 1
 void platform_init(int argc, char **argv);
+void platform_pace_poll( void );
```

Comment:
please remove the extra spaces around the void in the brackets - clang-format would remove them too.